app/main/templates/series_view.html

```diff
@@ -4,82 +4,97 @@
 {% block title %}{{title}}{% endblock %}
 
 {% block app_content %}
-    <!-- Series description -->
-    <div class="row">
-        <h2 class="text-center">DICOM series overview</h2>
-        <table class="table table-striped"> <!-- Series information -->
-            <tr>
-                <th>Series description</th>
-                <td>{{ series.description }}</td>
-            </tr>
-            <tr>
-                <th>Acquisition date</th>
-                <td>{{ series.series_datetime }}</td>
-            </tr>
-            <tr>
-                <th>Uploaded date</th>
-                <td>{{ series.created_at.format('YYYY-MM-DD HH:mm') }}</td>
-            </tr>
-            <tr>
-                <th>Files in series</th>
-                <td>{{ series.series_files }}</td>
-            </tr>
-            <tr>
-                <th>Reports made</th>
-                <td>{{ series.has_report }}</td>
-            </tr>
-        </table>
-        </br>
+
+    <div class="container py-5">
+        <div class="row">
+            <h1 class="h1 text-center justify-content-center">DICOM Series Overview</h1>
+        </div>
     </div>
 
-    <!-- Select new process -->
-    <div class="row">
-        <h4 class="text-left">Select measurement to be performed</h4>
-        <p>Hazenlib version: {{ hazenlib_version }}</p></br>
-        {% if form %}
-            {{ wtf.render_form(form) }}
-        {% endif %}
+    <div class="hazen-dotted-horizontal-line mb-3"></div>
+
+    <!-- Series Overview Table -->
+    <div class="container">
+        <div class="row pt-3">
+            <table class="table table-striped"> <!-- Series information -->
+                <tr>
+                    <th>Series description</th>
+                    <td>{{ series.description }}</td>
+                </tr>
+                <tr>
+                    <th>Acquisition date</th>
+                    <td>{{ series.series_datetime }}</td>
+                </tr>
+                <tr>
+                    <th>Uploaded date</th>
+                    <td>{{ series.created_at.format('YYYY-MM-DD HH:mm') }}</td>
+                </tr>
+                <tr>
+                    <th>Files in series</th>
+                    <td>{{ series.series_files }}</td>
+                </tr>
+                <tr>
+                    <th>Reports made</th>
+                    <td>{{ series.has_report }}</td>
+                </tr>
+            </table>
+        </div>
     </div>
 
-    <!-- Series results by task -->
+    <!-- Select New Process -->
+    <div class="row py-5">
+        <div class="col-md-6 center-block mx-auto">
+            <h4 class="h4">Select HAZEN task to perform:</h4>
+            <p class="p-front-page">hazenlib version: {{ hazenlib_version }}</p>
+            {% if form %}
+                {{ wtf.render_form(form) }}
+            {% endif %}
+        </div>
+    </div>
+
+    <div class="hazen-dotted-horizontal-line mb-3"></div>
```

Comment:
I would move this separator after the series info table and above the new task form